Sources/MockoloSupportRxSwift/PropertyMockable.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  Copyright (c) 2018. Uber Technologies
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+/**
+ The following property wrappers may be used for RxSwift observable variables to be mocked.
+
+ E.g.
+ public class FooMock: Foo {
+    @MockObservable  public var bazStream: Observable<Int> { get }
+ }
+ */
+
+#if canImport(RxSwift)
```

Comment:
`canImport` is not work for user defined packages. This is for platform dependent packages.